os/include/sys/socket.h

```diff
@@ -86,6 +86,88 @@ extern "C" {
 #define EXTERN extern
 #endif
 
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/* The socket()domain parameter specifies a communication domain; this selects
+ * the protocol family which will be used for communication.
+ */
+
+/* Supported Protocol Families */
+
+#ifndef PF_UNSPEC
+#define PF_UNSPEC 0 /* Protocol family unspecified */
+#endif
+#ifndef PF_UNIX
+#define PF_UNIX 1 /* Local communication */
+#endif
+#ifndef PF_LOCAL
+#define PF_LOCAL 1 /* Local communication */
+#endif
+#ifndef PF_INET
+#define PF_INET 2 /* IPv4 Internet protocols */
+#endif
+#ifndef PF_INET6
+#define PF_INET6 10 /* IPv6 Internet protocols */
+#endif
+#ifndef PF_NETLINK
+#define PF_NETLINK 16 /* Netlink IPC socket */
+#endif
+#ifndef PF_ROUTE
+#define PF_ROUTE PF_NETLINK /* 4.4BSD Compatibility*/
+#endif
+#ifndef PF_PACKET
+#define PF_PACKET 17 /* Low level packet interface */
+#endif
+#ifndef PF_BLUETOOTH
+#define PF_BLUETOOTH 31 /* Bluetooth sockets */
+#endif
+#ifndef PF_IEEE802154
+#define PF_IEEE802154 36 /* Low level IEEE 802.15.4 radio frame interface */
+#endif
+#ifndef PF_PKTRADIO
+#define PF_PKTRADIO 64 /* Low level packet radio interface */
```

Comment:
I think you'd better to remove macros (ex. PF_PKTRADIO ) which are not provided by TizenRT

Reply:
I don't know whether a PF is provided, So I remove PF which is not used in TizenRT including `external`.